docs/PID tuning.md

```diff
@@ -2,13 +2,11 @@
 
 Every aspect of flight dynamics is controlled by the selected "PID controller". This is an algorithm which is responsible for reacting to your stick inputs and keeping the craft stable in the air by using the gyroscopes and/or accelerometers (depending on your flight mode).
 
-The "PIDs" are a set of tuning parameters which control the operation of the PID controller. The optimal PID settings are different on every craft, so if you can't find someone with your exact setup who will share their settings with you, some trial and error is required to find the best performing PID settings.
+The "PIDs" is a set of tuning parameters that control the operation of the PID controller. The optimal PID settings are different on every craft, so if you can't find someone with your exact setup who will share their settings with you, some trial and error are required to find the best performing PID settings.
```

Comment:
I'd argue that `The "PIDs" is a set of tuning parameters that control` is less grammatically correct than `The "PIDs" are a set of tuning parameters which control.
And `some trial and error are required` is just plain incorrect.

Reply:
fixed.